html/screensharingtest.js

```diff
@@ -147,7 +147,7 @@ $(document).ready(function() {
 														{
 															// We want sendonly audio and screensharing
 															tracks: [
-																{ type: 'audio', capture: true, recv: false },
+																{ type: 'desktopAudio', capture: true, recv: false },
```

Comment:
I don't think this is a good idea, in the demo, or at the very least it should be optional and/or configurable. At least some time ago, desktop audio was only supported on Windows, and not other systems, which means you thought you asked for something and got something else entirely. This is why we fell back to "regular" audio in this demo, as it was meant as a way to "talk on top of your screen".

Reply:
Currently Chrome/Edge on windows can share audio in all modes (tab, window, screen) as long as the user ticks the permission box in the share screen prompt (tick for audio only appears if you request audio in the getDisplayMedia)

For other systems, Chrome/Edge can still share audio, but only of tabs.